src/app/components/LiveLabel/index.tsx

```diff
@@ -65,6 +65,8 @@ const LiveLabel = ({
             focusable="false"
             aria-hidden="true"
             viewBox="0 0 32 32"
+            width="1em"
```

Comment:
Not sure if you can put em here?

Reply:
It's used here in the webcore example seen here [https://github.com/bbc/web/blob/main/packages/design-system/components/live-pulse/live-pulse.jsx#L60](https://github.com/bbc/web/blob/main/packages/design-system/components/live-pulse/live-pulse.jsx#L60) so I reinstated it to prevent the svg from becoming huge when css is disabled. 😵‍💫